lib/dependencies/DepBlockHelpers.js

```diff
@@ -2,41 +2,33 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var DepBlockHelpers = exports;
+"use strict";
 
-DepBlockHelpers.getLoadDepBlockWrapper = function(depBlock, outputOptions, requestShortener, name) {
-	var promiseCode = DepBlockHelpers.getDepBlockPromise(depBlock, outputOptions, requestShortener, name);
+const DepBlockHelpers = exports;
+
+DepBlockHelpers.getLoadDepBlockWrapper = (depBlock, outputOptions, requestShortener, name) => {
+	const promiseCode = DepBlockHelpers.getDepBlockPromise(depBlock, outputOptions, requestShortener, name);
 	return [
 		promiseCode + ".then(",
 		").catch(",
 		")"
 	];
 };
 
-DepBlockHelpers.getDepBlockPromise = function(depBlock, outputOptions, requestShortener, name) {
+DepBlockHelpers.getDepBlockPromise = (depBlock, outputOptions, requestShortener, name) => {
 	if(depBlock.chunks) {
-		var chunks = depBlock.chunks.filter(function(chunk) {
-			return !chunk.hasRuntime() && chunk.id !== null;
-		});
+		const chunks = depBlock.chunks.filter(chunk => !chunk.hasRuntime() && chunk.id !== null);
 		if(chunks.length === 1) {
-			var chunk = chunks[0];
-			return "__webpack_require__.e" + asComment(name) + "(" + JSON.stringify(chunk.id) + "" +
-				(outputOptions.pathinfo && depBlock.chunkName ? "/*! " + requestShortener.shorten(depBlock.chunkName) + " */" : "") +
-				asComment(depBlock.chunkReason) + ")";
+			const chunk = chunks[0];
+			return `__webpack_require__.e${asComment(name)}(${JSON.stringify(chunk.id)}${outputOptions.pathinfo && depBlock.chunkName ? "/*! " + requestShortener.shorten(depBlock.chunkName) + " */" : ""}${asComment(depBlock.chunkReason)})`;
```

Comment:
Try to move parts of the expression into a separate variable to avoid that long lines